MAINTAINERS

```diff
@@ -0,0 +1,19 @@
+# The official list of maintainers, reviewers and members for the project maintenance.
+#
+# Refer to the GOVERNANCE.md in etcd repository for description of the roles.
+#
+# Names should be added to this file like so:
+#     Individual's name <submission email address> (@GITHUB_HANDLE) pkg:*
+#     Individual's name <submission email address> <email2> <emailN> (@GITHUB_HANDLE) pkg:*
+#
+# Please keep the list sorted.
+
+# MAINTAINERS
+Benjamin Wang <wachao@vmware.com> (ahrtr@) pkg:*
+Marek Siarkowicz <marek.siarkowicz@gmail.com> (@serathius) pkg:*
+Piotr Tabor <ptab@google.com> (@ptabor) pkg:*
+
+# REVIEWERS
+
+# MEMBERS
```

Comment:
Don't think members need a special section. As mentioned in https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md#member members are defined by being added to an etcd.io org.

Reply:
thx & removed.